SQL/New_patches/2019-11-01-Add_data_release_permissions.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO permissions (code,description,categoryID) VALUES 
+('data_release_view','Data Release: View releases',(SELECT ID FROM permissions_category WHERE Description='Permission'));
+
+INSERT INTO LorisMenuPermissions (MenuID, PermID)
+    SELECT m.ID, p.PermID FROM permissions p CROSS JOIN LorisMenu m WHERE p.code='data_release_view' AND m.Label='Data Release';
+
+INSERT IGNORE INTO user_perm_rel
+SELECT upr.userID, p.permID FROM user_perm_rel upr JOIN permissions p WHERE upr.permID=1 AND p.code IN ('data_release_view');
```

Comment:
Does this line give the permission to the first user in the SQL table aka admin?

Reply:
yes